src/Digdir.Tool.Dialogporten.MigrationVerifier/Program.cs

```diff
@@ -0,0 +1,14 @@
+using Digdir.Tool.Dialogporten.MigrationVerifier;
+using Microsoft.ApplicationInsights.Extensibility;
+using Serilog;
+
+Log.Logger = new LoggerConfiguration()
+    .MinimumLevel.Information()
+    .Enrich.FromLogContext()
+    .WriteTo.ApplicationInsights(
+        TelemetryConfiguration.CreateDefault(),
+        TelemetryConverter.Traces)
+    .CreateBootstrapLogger();
+
+await MigrationVerifier.Verify(Log.Logger);
```

Comment:
Bør vi returnere en eller annen form for feilkode her til shellet?

Reply:
Tror ikke vi trenger det?

Vi logger/kaster exceptions i Verify(), som resulterer i at vi får en ikke-0-exit
Hvis dotnet avslutter pent av seg selv får vi exit 0